app/js/main.js

```diff
@@ -725,9 +758,11 @@ function _writeFile(file, data) {
     "use strict";
      fs.writeFile(file, data, function(err) {
         if (err) {
-            throw err;
+            console.error(`Failed to save ${file} to disk`);
+            createSaveDirectory();
```

Comment:
I'm not so sure about this call. It's assuming the error was caused by a missing output folder, which is not always the case. We also talked about the unlikelyhood of the output folder being ripped out from under the app.

I'll leave it up to do. Do what you think is best.